accord-core/src/main/java/accord/local/SafeCommandStore.java

```diff
@@ -303,6 +305,11 @@ public void updateMaxConflicts(Command prev, Command updated)
         commandStore().updateMaxConflicts(prev, updated);
     }
 
+    public long maxHLCForSeekables(Routables<?> keysOrRanges)
```

Comment:
simply `maxHlc`? You're accepting `Routables` not `Seekables` and including the type isn't probably necessary anyway. Mostly we have lower cased the `lc` in `Hlc` so far in the codebase.

Reply:
Unused, removed